genesyscloud/provider/provider.go

```diff
@@ -244,10 +245,66 @@ func InitClientConfig(data *schema.ResourceData, version string, config *platfor
 	oauthclientID := data.Get("oauthclient_id").(string)
 	oauthclientSecret := data.Get("oauthclient_secret").(string)
 	basePath := GetRegionBasePath(data.Get("aws_region").(string))
+	config.BasePath = basePath
 
-	sdkDebugFilePath := data.Get("sdk_debug_file_path").(string)
+	diagErr := setUpSDKLogging(data, config)
```

Comment:
I moved the sdk logging setup and proxy setup to their own file because this function was getting very big